deployment/app.py

```diff
@@ -0,0 +1,476 @@
+# -*- coding: utf-8 -*-
+from flask import Flask, jsonify, send_file, request
+import turicreate as tc
+from enum import Enum
+from PIL import Image
+import numpy as np
+import argparse
+import datetime
+import zipfile
+import json
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+class ModelType(Enum):
```

Comment:
I suggest removing the ModelType enum, and the info dict. You should be able to streamline a lot of this logic by using the `model` object directly (which comes with all the same info as is in the `info` dict, including its type information).